lerna.js

```diff
@@ -104,3 +104,33 @@ async function depcheckTask(log) {
 
 module.exports.depfixTask = depfixTask
 module.exports.depcheckTask = depcheckTask
+
+const ncu = require('npm-check-updates');
```

Comment:
This script seems really valuable!
Perhaps we want to run it in a separate PR, so it doesn't pollute this one too much?

Reply:
Sure - only problem is that it bumps AsyncIterator to 3.6.1 which breaks some unit tests because of the `_fillBuffer` usage so the fixes here will need to be merged first.